### PR TITLE
RTFHandler: fixed problem with plain chars from no-ascii charset

### PR DESCRIPTION
**Pull request by javierag**
Also I have added two commits with defensive enhancements.

### CODE REVIEW

**Comment by blaxter** [comment]
Typo on 080465c commit message

File: OpenChange/RTFHandler.m [hunk 1]
```diff
@@ -1437,20 +1450,17 @@ - (NSMutableData *) parse
         {
           c = *_bytes;
           // We avoid appending NULL bytes or endlines
-          if (c && (c != '\n')) 
+          if (c && (c != '\n') && (c != '\r'))
             {
-              if (c <= UTF8_FIRST_BYTE_LAST_CODEPOINT)
```
Comment (blaxter) [inline OpenChange/RTFHandler.m]: Now `UTF8_FIRST_BYTE_LAST_CODEPOINT` is no longer used. We should remove the `define`

**Comment by blaxter** [comment]
Apart from those 2 minor things :ok_hand:

**Comment by javierag** [comment]
Thanks for the comments @blaxter  . I have fixed this tow things and also removed a define from a previous version of the fix.

**Comment by blaxter** [comment]
:+1:

**Comment by sixstone-qq** [comment]
:8ball: :100:

**Pull request merged**